app/src/main/java/org/oppia/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -20,6 +20,7 @@ import org.oppia.app.viewmodel.ViewModelProvider
 import org.oppia.domain.profile.ProfileManagementController
 import javax.inject.Inject
 
+private const val USER_NAME_MAX_CHAR_COUNT = 36
```

Comment:
This constant appears unused, please remove it.

Reply:
Done.